### PR TITLE
use flavour_ram instead of id

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
Replace flavour id with ram, see http://docs.ansible.com/ansible/nova_compute_module.html

thx @manics for a hint

### CODE REVIEW

**Comment by aleksandra-tarkowska** [comment]
https://10.0.51.146:8443 has 4 cores 16 GB allocated but OMERO-build job is slower (32 GB or 24GB/8CPU - 13m, 16GB/4CPU - 20m), will let you know about tests when they finishes

maybe we could do something in the middle like 16GB and 6CPU

File: ansible/os-devspace.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   vars:
 
     vm_image: "CentOS 7"
-    vm_flavour: m2.large
+    vm_flavour_ram: 24576
```
Comment (manics) [inline ansible/os-devspace.yml]: a nitpicky suggestion, use 24000 in case people use units of 1000 instead of 1024 (since this is a minimum requirement)

**Comment by aleksandra-tarkowska** [comment]
Interesting that devspace built yesterda, although today BF job 

```
[testng] Java HotSpot(TM) 64-Bit Server VM warning: INFO: os::commit_memory(0x000000066af80000, 172490752, 0) failed; error='Cannot allocate memory' (errno=12)
```

@sbesson what you think? do we actually need 32GB?

**Comment by sbesson** [comment]
@aleksandra-tarkowska: sorry for the delay. After independent retesting, I was able to reproduce the issue you reported once after 8 independent builds (https://10.0.51.143:8443/job/BIOFORMATS-ant/13/) and have retrievedthe `hs_err_pid17045.log` locally for anyone interested in debugging. Following this, tried to relaunch the same job with a lower number of writing tests seems to be help letting the build remaining reliable.

Thinking of a compromise  to allow the reduction of unnecessary memory while maintaining minimal coverage, I would suggest we drop https://github.com/openmicroscopy/devspace/pull/63/commits/a98d2422f465b8a849dbb23b41edf5f72a310a1b from the devspace PR especially as a minimal number of writing tests are now be executed by default on the `develop` branch and re-include BIOFORMATS-ant in the Trigger build. Thoughts?

**Comment by aleksandra-tarkowska** [comment]
as discussed we will reduce https://github.com/openmicroscopy/devspace/pull/63/commits/a98d2422f465b8a849dbb23b41edf5f72a310a1b to default values in https://github.com/openmicroscopy/bioformats/pull/2717, if someone wish to increase them just need more ram

File: ansible/roles/devspace/tasks/devspace-install.yml [hunk 2]
```diff
@@ -119,7 +120,6 @@
     regexp: '^ARG USER_ID=1000'
     replace: 'ARG USER_ID={{ devspace_user_id }}' # how to find out UID?
   with_items:
-    - bf/Dockerfile
```
Comment (sbesson) [inline ansible/roles/devspace/tasks/devspace-install.yml]: Should I close https://github.com/openmicroscopy/infrastructure/pull/235 in favor of this PR?

Comment (aleksandra-tarkowska) [inline ansible/roles/devspace/tasks/devspace-install.yml]: sorry I haven't notice. yes please

**Comment by joshmoore** [comment]
This needs bumping of the devspace version: https://github.com/openmicroscopy/devspace/releases/tag/0.4.0

**Comment by aleksandra-tarkowska** [comment]
Done, played on 

```
$ ansible-playbook -l devspace-test-123 -u centos devspace.yml -vvv 
...

PLAY RECAP *********************************************************************
7c757072-6a62-48b2-aa97-1a9a1bb95731 : ok=37   changed=28   unreachable=0    failed=0   
```

**Comment by sbesson** [comment]
:+1: @joshmoore any veto to merging? Else I'll get this in and test it independently once I fixed the conflicts in https://github.com/openmicroscopy/devspace/pull/59

**Comment by joshmoore** [comment]
:+1:

**Pull request merged**